dashboard/config/routes.rb

```diff
@@ -644,6 +644,8 @@
     end
   end
 
+  get 'feedback', to: 'feedback#index'
```

Comment:
I've got some interest in doing

```suggestion
  resources :feedback
```

instead, leaning on Rails convention, and maybe even collapsing some of our `/api/v1/...` routes into this resource eventually.  Open to pushback on this though - I might be overthinking it.